weave/trace_server/trace_server_interface.py

```diff
@@ -51,6 +51,8 @@ class CallSchema(BaseModel):
 
     deleted_at: typing.Optional[datetime.datetime] = None
 
+class CallSchemaWithCosts(CallSchema):
+    costs: typing.Optional[typing.Dict[str, typing.Any]] = None
```

Comment:
you might just want to put costs in the `summary` dictionary under a `[weave][costs]` key. This would avoid messing with the underlying data model and keep usage / costs co-located.

Reply:
that does make more sense